mibidata/mibi_image.py

```diff
@@ -307,7 +307,14 @@ def __str__(self):
         return s
 
     @staticmethod
-    def validate_aperture(value):
+    def parse_aperture(value):
+        """
+        Args:
+            value: A string representing an aperture code.
+        Returns:
+            an aperture code (e.g. 'A' or 'B') matching aperture width
+            used during image acquisition.
```

Comment:
Please include a `Raises` section in the docstring.

Reply:
Done.